.github/workflows/release.yml

```diff
@@ -26,6 +26,6 @@ jobs:
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
       - run: git update-index --assume-unchanged .npmrc
-      - run: npx lerna publish from-package --yes
+      - run: npx nx release publish
```

Comment:
I tried this locally with `--dry-run` and it seemed to work, but we won't know for sure until we try to publish a release.